rts/Rendering/Fonts/CFontTexture.cpp

```diff
@@ -74,12 +74,40 @@
 typedef unsigned char FT_Byte;
 #endif
 
+constexpr int MAX_TEMPORAL_FONTS = 10;
```

Comment:
Could use an explanation.
```suggestion
/* Stores the most recently used fallback fonts, otherwise they get
 * evicted from cache and need to be loaded all over again. */
constexpr int MAX_TEMPORAL_FONTS = 10;
```
Or perhaps just "TEMPORAL" -> "RECENTLY_USED"